src/Lorentz/Contracts/BaseDAO/Doc.hs

```diff
@@ -0,0 +1,168 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+module Lorentz.Contracts.BaseDAO.Doc
+   ( introductoryDoc
+
+   , callFA2Doc
+   , transferDoc
+   , balanceOfDoc
+   , tokenMetadataRegistryDoc
+   , updateOperatorsDoc
+
+   , transferOwnershipDoc
+   , acceptOwnershipDoc
+   , migrateDoc
+   , confirmMigrationDoc
+
+   , proposeDoc
+   , voteDoc
+   , setVotingPeriodDoc
+   , setQuorumThresholdDoc
+   , flushDoc
+
+   , burnDoc
+   , mintDoc
+   , transferContractTokensDoc
+   , tokenAddressDoc
+   ) where
+
+import Lorentz
+
+import Util.Markdown
+
+introductoryDoc :: Markdown
+introductoryDoc = [md|
+  It contains standard FA2 entrypoints, plus some extra ones including proposal and
+  migration entrypoints. It supports two types of token_id - frozen(token_id = 1) and unfrozen(token_id = 0).
+  |]
+
+callFA2Doc :: Markdown
+callFA2Doc = [md|
+  Entrypoint to be called if you want to use one of FA2 entrypoints.
+  |]
+
+transferDoc :: Markdown
+transferDoc = [md|
+  Transfer tokens between a given account and each account from the given list.
+
+  It serves multiple purposes:
+  * If transaction `"from"` address equals to the admin address presented in storage, then  it is allowed for
+  any address and both `frozen` and `unfrozen` tokens
+  * Otherwise, it is allowed to transfer money only if from address equals to sender address or have sender as an operator
+  It is also prohibited to send frozen tokens in this case.
+  |]
+
+balanceOfDoc :: Markdown
+balanceOfDoc = [md|
+  Returns the balance of specified address in ledger.
+  The entrypoint supports both frozen and unfrozen tokens.
+  |]
+
+tokenMetadataRegistryDoc :: Markdown
+tokenMetadataRegistryDoc = [md|
+  Returns contract address that holds token metadata.
+  Token metadata MUST contain the DAO metadata.
```

Comment:
I think it is better to replace `must` with `will`, since unlike in the specification, here we describe our contract's guarantees.